components/Common/Breadcrumbs/index.tsx

```diff
@@ -0,0 +1,59 @@
+import classNames from 'classnames';
+import { type LinkProps } from 'next/link';
+import Link from 'next/link';
```

Comment:
Can you replace this by `LocalizedLink`, we avoid using `next/link` and we only use LocalizedLink which extends `next/link`